imprime_pares/spec/imprime_pares_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_base'
+require 'imprime_pares'
+
+describe 'Pares' do
+  it 'Retorne valor inválido quando o valor de entrada não for um número' do
```

Comment:
```suggestion
  it 'Retorne valor inválido quando o valor de entrada não for um inteiro' do
```
Aqui, "inteiro" não seria mais adequado, visto que "5" é um número, mas é um número em formato de string. O que acha?

Reply:
Boa!